lab-trent/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const http = require('http');
+
+const requestParser = require('./lib/requestparser');
+const cowsayRes = require('./lib/cowsayres');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer(function(req, res) {
+  requestParser.parseUrlData(req);
```

Comment:
I like that you separated this out into it's own function, but given that you're relying on side effects, it might be best to just have those couple lines right here inside of the create server callback.